src/main.rs

```diff
@@ -73,39 +96,35 @@ fn print_usage(program: &str, opts: Options) {
     print!("{}", opts.usage(&brief));
 }
 
-fn run_client(host: &str, port: u16, time: u64) -> Result<(), ::std::io::Error> {
-    let mut c = try!(client::TestClient::new(host, port));
+fn run_client(host: &str, port: u16, time: u64, pings: u64) -> Result<(), ::std::io::Error> {
+    let mut c = try!(test_client::TestClient::new(host, port));
 
-    print!("Testing ping... ");
+    info!("Testing ping... ({} times) ", pings);
     try!(stdout().flush());
-    println!("done, {:.*} ms", 2, try!(c.test_ping(20)));
+    let ping_time = try!(c.test_ping(pings));
+    info!("done, {:.*} ms", 2, ping_time);
 
-    print!("Testing download... ");
+    info!("Testing download... ({} seconds) ", time);
     try!(stdout().flush());
-    println!("done, {}", format_speed(try!(c.test_downstream(time * 1_000u64))));
+    let download_speed = try!(c.test_downstream(time * 1_000u64));
+    info!("done, {}", format_speed(download_speed));
 
-    print!("Testing upload... ");
+    info!("Testing upload... ({} seconds) ", time);
     try!(stdout().flush());
-    println!("done, {}", format_speed(try!(c.test_upstream(time * 1_000u64))));
+    let upload_speed = try!(c.test_upstream(time * 1_000u64));
+    info!("done, {}", format_speed(upload_speed));
 
     Ok(())
 }
 
-fn launch_server(port: u16, listen: &str) {
-    println!("Listening...");
-    let s = server::TestServer::new(port, listen);
-    s.listen();
-}
-
 fn format_speed(speed: f64) -> String {
     let mut speed = speed;
-    let units = ["bit/s", "kbit/s", "Mbit/s", "Gbit/s", "Tbit/s?!"];
+    let units = ["bit/s", "Kbit/s", "Mbit/s", "Gbit/s", "Tbit/s"];
```

Comment:
I actually checked and it's kbit. Also refer to https://en.wikipedia.org/wiki/Data_rate_units